pom.xml

```diff
@@ -45,6 +45,8 @@
         <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.project.outputEncoding>UTF-8</project.project.outputEncoding>
+
+        <jopt.simple.module.name>jopt.simple</jopt.simple.module.name>
```

Comment:
Also, what about joptsimple instead? To me, the . implies some sort of namespace.

Reply:
That's a good idea. Have changed to `joptsimple`

Reply:
If we didn't specify a module name, Java 9 would automatically generate a module name `jopt.simple` from the jar name `jopt-simple`.  So with this better name `joptsimple` specified, Java 9 will be able to recognise `joptsimple`.